_userdocs/Why-Choose-OpenStack.md

```diff
@@ -144,7 +144,7 @@ A few vendors offer OpenStack cloud through the hosted, or ‘as a service’ de
 
 Hosted cloud customers receive their single-tenant OpenStack cloud with all its advantages, including API access without the overhead of deployment and operations. A comprehensive list of OpenStack hosted cloud providers is available on the OpenStack Marketplace. Here are some popular vendors of OpenStack cloud as a service:
 
-**IBM Blue Box**: Blue Box, an IBM Company, was one of the earliest providers to offer hosted OpenStack private cloud as a service—PcaaS. The company was purchased by IBM in June, 2015 to become a part of IBM’s cloud product offering. Blue Box leads the OpenStack Seattle User Group meetups and actively contributes to the OpenStack community, particularly around OpenStack Operators work group. Their latest product, IBM Blue Box Cloud, lets you locate OpenStack private cloud hardware in your own data center, fully managed by IBM Blue Box experts.
+**IBM Bluemix on OpenStack**: Bluemix on OpenStack, formerly known as Blue Box, an IBM Company, was one of the earliest providers to offer hosted OpenStack private cloud as a service—PcaaS. The company was purchased by IBM in June, 2015 to become a part of IBM’s cloud product offering. Bluemix on OpenStack leads the OpenStack Seattle User Group meetups and actively contributes to the OpenStack community, particularly around OpenStack Operators work group. Their latest product, IBM Bluemix on OpenStack, lets you locate OpenStack private cloud hardware in your own data center, fully managed by IBM Bluemix on OpenStack experts.
```

Comment:
Change this phrase to read as follows:

**IBM Bluemix on OpenStack**: Blue Box, now part of IBM and known as IBM Bluemix on OpenStack, was one of the earliest providers to offer hosted OpenStack private cloud as a service—PcaaS. The company was purchased by IBM in June, 2015 to become a part of IBM’s cloud product offering. The team at IBM Bluemix on OpenStack leads the OpenStack Seattle User Group meetups and actively contributes to the OpenStack community, particularly around OpenStack Operators work group. Their latest product, IBM Bluemix Local on OpenStack, lets you locate OpenStack private cloud hardware in your own data center, fully managed by our IBM Bluemix on OpenStack experts.